release_notes/ocp_3_10_release_notes.adoc

```diff
@@ -54,33 +54,524 @@ documentation.
+
+[[ocp-310-system-services-now-hosted-on-pods]]
+==== System Services Now Hosted on Pods
+Each of the system services, API, controllers, and etcd, used to run as system
+services on the master. These services now run on static pods in the cluster. As
+a result, there are new commands to restart these services: `master-restart
+api`, `master-restart controllers`, and m`aster-restart etcd`. To view log
+information on these services, use `master-logs api api`, `master-logs
+controllers controllers`, and `master-logs etcd etcd`.
+
+[[ocp-310-new-node-configuration-process]]
+==== New Node Configuration Process
+You can modify exiting nodes through a configuration map rather than the
+*_node-config.yaml_*. The installation creates three node configuration groups:
+*node-config-master*, *node-config-infra*, and *node-config-compute* and creates
+a configuration map for each group.  A sync pod watches for changes to these
+configuration maps. When a change is detected, the sync pod updates the
+*_node-config.yaml_* file on all of the nodes.
+
+[[ocp-310-group-pruning]]
+==== Group Pruning
+To prune groups records from an external provider, administrators can
+run the following command:
+
+----
+$ oc adm prune groups --sync-config=path/to/sync/config [<options>]
+----
+See xref:../admin_guide/pruning_resources.adoc#pruning-groups[Pruning groups] for more information.
+
 [[ocp-310-installation]]
 === Installation
 
+[[ocp-310-important-installation-changes]]
+==== Important Changes
+
+A number of important changes have been made to the installation process in
+{product-title} 3.10.
+
+* The control plane components (etcd, API server, and controllers) are now run as
+static pods by the kubelet on the masters. This is now the only supported way
+they can be run. The upgrade will change over to this automatically. You can see
+the pods once the control plane comes up with `oc get pods -n kube-system` and
+execute, log, and inspect them using normal `oc` CLI commands.
+
+* The containerized mode for {product-title} is no longer supported (where
+ components run in docker containers) and the 3.10 upgrade will require you to
+ move to RPMs or system containers for the kubelet by setting
+ `openshift_use_system_containers=true`.
+
+* System containers for the control plane components are no longer supported.
+Those components run as static pods instead and the upgrade will automatically
+make this conversion.
+
+* Node bootstrapping (controlled by the inventory variable
+`openshift_node_bootstrap`) defaults to `True` instead of `False`, which means nodes
+will pull their configuration and client and server certificates from the
+master. The 3.10 upgrade, unless you set `openshift_node_bootstrap=false` in your
+inventory, will automatically transform your nodes to use that mode.  This
+enables certificate rotation and centralized certificate management by default. Use `oc get
+csr` to see pending certificates.
```

Comment:
`openshift_node_bootstrap` is also removed, please see https://bugzilla.redhat.com/show_bug.cgi?id=1567783.

All nodes must be bootstrapped.